Makefile

```diff
@@ -19,6 +20,7 @@ upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -q -r requirements/pip_tools.txt
 	pip-compile --upgrade -o requirements/pip_tools.txt requirements/pip_tools.in
+	pip-compile --rebuild --upgrade --allow-unsafe -o requirements/pip.txt requirements/pip.in
```

Comment:
The `--rebuild` option should be on the first pip-compile run, and you added this below the one for `pip_tools.in`.

Reply:
Made the change as per suggestion.